### ISSUE TITLE
Trakt not syncing & not hiding watched seasons > completely watched (all shows within all seasons) shows

### ISSUE THREAD

**Issue by splaquet** [elgatito/elementum#22]
The subject pretty much says it all. The odd part is that it will hides some completely watched show seasons. It seems to mostly recognized watched seasons if the Elementum install & sync happens after they’ve already been synced. It does not seem to recognize completely watched shows, when they’ve been completed (or marked as completed) after the install/after the first initial sync. 

I might be mistaken, but I believe this functionality worked as expected before the v5 major version upgrade. Hasn’t appeared to work as expected since v5+

**Comment by I-Cat** [comment]
Ya I have been working on importing my Trakt tracker.

**Comment by Janhouse** [comment]
It's funny that one of the Trak developers reached out to me today and asked if I could try to remove duplicates from my Trakt profile. When I asked which integration is making duplicates, I got this message: "[..]It looks like it’s entirely Elementum. I did a little more digging and it looks like you actually only have like 7000 records, but Elementum is adding and deleting a ton of records a day.  You’re up to 3.8m activities[..]".

Elementum has currently broken my Trakt profile, and I can't even access their web. :smile:
